internal/k8shandler/reconciler.go

```diff
@@ -124,6 +127,27 @@ func Reconcile(requestCluster *elasticsearchv1.Elasticsearch, requestClient clie
 		ll:       log.WithValues("cluster", requestCluster.Name, "namespace", requestCluster.Namespace),
 	}
 
+	// TODO: check if we are doing ES cert management looking for annotation:
+	// logging.openshift.io/elasticsearch-cert-management: true
+	value, ok := requestCluster.Annotations[constants.EOCertManagementLabel]
+	if ok {
+		manageBool, _ := strconv.ParseBool(value)
+		if manageBool {
+			cr := NewCertificateRequest(requestCluster.Name, requestCluster.Namespace, requestCluster.GetOwnerRef(), requestClient)
+			cr.GenerateElasticsearchCerts(requestCluster.Name)
+
+			// for any components specified like:
+			// logging.openshift.io/elasticsearch-cert/{secret_name}: {component_name}
```

Comment:
Mismatch between note and prefix and proposal

Reply:
thanks, i'll update the comment. proposal was updated